server/resource.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"net/url"
 	"strconv"
+        "strings"
```

Comment:
This line is prefixed by a few spaces, instead of a tab. Replace it and the check should succeed!